src/main/tools/pmd/pmd-rules.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0"?>
```

Comment:
Started without making this a child of freelib, thus the version numbers in the pom.  Switched to making it a child project because checkstyle wasn't getting our style.  Expect the pmd-rules was created before making the child-project switch. I'll delete locally and see if it gets rebuilt.